web/src/main/java/com/epam/rft/atsy/web/model/file/FileStatus.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.rft.atsy.web.model.file;
+
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public enum FileStatus {
+
+  FILE_IS_NOT_EXIST(false),
+  FILE_IS_IN_PROGRESS(null),
+  FILE_IS_ALREADY_EXIST(true);
+
+  public static final String CV_STATUS = "cv_status";
```

Comment:
You could just return the enum values and check them in the jsp, a boolean is not too good here and not needed I think.

Also the name should contain info about it's purpose. E.g. CV_STATUS_MODEL_ATTR_NAME